src/objecttypes/api/serializers.py

```diff
@@ -110,6 +110,7 @@ class Meta:
             "providerOrganization",
             "documentationUrl",
             "labels",
+            "linkable_to_zaken",
```

Comment:
Overlooked this last time :facepalm:, but since all the other fields are camelcased, let's make sure this one is as well

```suggestion
            "linkableToZaken",
```